workitem/simple_type.go

```diff
@@ -38,12 +38,12 @@ func (t SimpleType) GetKind() Kind {
 var timeType = reflect.TypeOf((*time.Time)(nil)).Elem()
 
 // ConvertToModel implements the FieldType interface
-func (fieldType SimpleType) ConvertToModel(value interface{}) (interface{}, error) {
+func (t SimpleType) ConvertToModel(value interface{}) (interface{}, error) {
```

Comment:
Why do you make this change in this PR? It has nothing to do with the topic of this PR.